### PR TITLE
Make debugging reservation time configurable.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: src/rest-server/src/config/paiConfig.js [hunk 1]
```diff
@@ -32,12 +32,14 @@ try {
...
 let paiConfigData = {
     machineList: paiMachineList,
+    debuggingReservationSeconds: process.env.DEBUGGING_RESERVATION_SECONDS,
 };
 
 
 // define the schema for pai configuration
 const paiConfigSchema = Joi.object().keys({
     machineList: Joi.array(),
+    debuggingReservationSeconds: Joi.number().integer().positive(),
```
Comment (Gerhut) [inline src/rest-server/src/config/paiConfig.js]: Shall this validation pass without `Number(process.env.DEBUGGING_RESERVATION_SECONDS)` convertion in line 35?

Comment (ydye) [reply src/rest-server/src/config/paiConfig.js]: Done

File: src/rest-server/src/config/paiConfig.js [hunk 2]
```diff
@@ -32,12 +32,14 @@ try {
 
 let paiConfigData = {
     machineList: paiMachineList,
+    debuggingReservationSeconds: Number(process.env.DEBUGGING_RESERVATION_SECONDS),
```
Comment (Gerhut) [inline src/rest-server/src/config/paiConfig.js]: To make the rest-server could be run separately, I think it's better to give a default in this stage.
```suggestion
    debuggingReservationSeconds: Number(process.env.DEBUGGING_RESERVATION_SECONDS || 604800),
```

Comment (ydye) [reply src/rest-server/src/config/paiConfig.js]: Done

**Pull request merged**